src/main/java/edu/harvard/hul/ois/fits/tools/embarc/EmbARC.java

```diff
@@ -0,0 +1,351 @@
+		String colorSpace = mapDescriptorToColorSpace(getValueForColumn(metadata, DPXColumn.DESCRIPTOR_1));
+		String linesPerImageElement = getValueForColumn(metadata, DPXColumn.LINES_PER_IMAGE_ELEMENT);
+		String creator = getValueForColumn(metadata, DPXColumn.CREATOR);
+		String pixelsPerLine = getValueForColumn(metadata, DPXColumn.PIXELS_PER_LINE);
+		String orientation = mapImageOrientationToOrientation(getValueForColumn(metadata, DPXColumn.IMAGE_ORIENTATION));
+		String inputDeviceName = getValueForColumn(metadata, DPXColumn.INPUT_DEVICE_NAME);
+		String inputDeviceSerialNumber = getValueForColumn(metadata, DPXColumn.INPUT_DEVICE_SERIAL_NUMBER);
+
+		if (bitDepth != null && !bitDepth.isEmpty()) {
+			addElement(imageElement, FitsMetadataValues.BITS_PER_SAMPLE, bitDepth);
+		}
+
+		if (byteOrder != null && !byteOrder.isEmpty()) {
+			addElement(imageElement, FitsMetadataValues.BYTE_ORDER, byteOrder);
+		}
+
+		if (colorSpace != null && !colorSpace.isEmpty()) {
+			addElement(imageElement, FitsMetadataValues.COLOR_SPACE, colorSpace);
+		}
+
+		if (linesPerImageElement != null && !linesPerImageElement.isEmpty()) {
+			addElement(imageElement, FitsMetadataValues.IMAGE_HEIGHT, linesPerImageElement);
+		}
+
+		if (pixelsPerLine != null && !pixelsPerLine.isEmpty()) {
+			addElement(imageElement, FitsMetadataValues.IMAGE_WIDTH, pixelsPerLine);
+		}
+
+		if (creator != null && !creator.isEmpty()) {
+			addElement(imageElement, FitsMetadataValues.IMAGE_PRODUCER, creator);
+		}
+
+		if (orientation != null && !orientation.isEmpty()) {
+			addElement(imageElement, FitsMetadataValues.ORIENTATION, orientation);
+		}
+
+		if (inputDeviceName != null && !inputDeviceName.isEmpty()) {
+			addElement(imageElement, FitsMetadataValues.SCANNER_MODEL_NAME, inputDeviceName);
+		}
+
+		if (inputDeviceSerialNumber != null && !inputDeviceSerialNumber.isEmpty()) {
+			addElement(imageElement, FitsMetadataValues.SCANNER_MODEL_SERIAL_NO, inputDeviceSerialNumber);
+		}
+
+		metadataElement.addContent(imageElement);
+		return metadataElement;
+	}
+
+	private Document createRawDataXml(DPXFileInformation dpxFileInfo) throws FitsToolException {
+		StringWriter out = new StringWriter();
+		out.write("<?xml version=\"1.0\" encoding=\"UTF-8\"?>");
+		out.write("\n");
+		out.write("<embARC>");
+		out.write("\n");
+		out.write("<rawOutput>\n");
+		String dpxXml = XmlWriterDpx.createDpxMetadataXml(dpxFileInfo);
+		if (dpxXml != null) {
+			out.write(dpxXml);
+		}
+		out.write("</rawOutput>");
+		out.write("\n");
+		out.write("</embARC>");
+		out.write("\n");
+		out.flush();
+
+		try {
+			out.close();
+		} catch (IOException e) {
+			throw new FitsToolException("Error closing embARC tool XML output stream");
+		}
+
+		Document doc = null;
+		try {
+			doc = saxBuilder.build(new StringReader(out.toString()));
+		} catch (Exception e) {
+			throw new FitsToolException("Error parsing embARC tool XML raw output");
+		}
+
+		return doc;
```

Comment:
This could be simplified to:

```java
		try {
                    return saxBuilder.build(new StringReader(out.toString()));
		} catch (Exception e) {
			throw new FitsToolException("Error parsing embARC tool XML raw output");
		}
```